wallet-webapps-common/src/main/webapp/vue-app/wallet-settings/components/WalletSettings.vue

```diff
@@ -63,15 +78,24 @@ export default {
     this.$nextTick().then(() => this.$root.$applicationLoaded());
   },
   methods: {
+    init() {
+      this.walletUtils.initSettings(eXo.env.portal.spaceName !== '', true)
+        .then(() => this.walletUtils.initWeb3(this.isSpace, true));
+    },
     openDetail() {
       document.dispatchEvent(new CustomEvent('hideSettingsApps', {detail: this.id}));
       this.getWallet();
       this.displayDetails = true;
+      this.init();
     },
     closeDetail() {
       document.dispatchEvent(new CustomEvent('showSettingsApps'));
       this.displayDetails = false;
-      window.history.replaceState('wallet', 'My wallet', `${eXo.env.portal.context}/${eXo.env.portal.portalName}/settings`);
+      if (this.from === 'space' || eXo.env.portal.spaceName !== ''){
+        window.history.replaceState('wallet', this.$t('exoplatform.wallet.title.myWallet'), `${eXo.env.portal.context}/g/:spaces:${eXo.env.portal.spaceGroup}/${eXo.env.portal.spaceName}/settings`);
```

Comment:
I believe we need another resource bundle key for the page title.

Reply:
done!